src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement/Controllers/SystemUserClientDelegationController.cs

```diff
@@ -135,6 +132,16 @@ public async Task<ActionResult> DeleteClientDelegation([FromQuery] Guid party, [
             return BadRequest("Party does not match delegation facilitator");
         }
 
+        // Redundant? This is Facilitator check...
```

Comment:
Nei, dette er vel ikke redundant. Om vi ikke sjekker at delegation som slettes er knytt til party/facilitator's klient-rolle så skal jo ikke Klientadministrator for party ha lov å slette delegeringen.